packages/api/src/server/storage/quote.ts

```diff
@@ -1,6 +1,11 @@
-import { IAsset, IAssetMapById, getAssetsMaps, isDevMode } from '@src/utils';
-import RedisReadClient from '@src/utils/redis/RedisReadClient';
-import RedisWriteClient from '@src/utils/redis/RedisWriteClient';
+import {
+  RedisReadClient,
+  RedisWriteClient,
+  IAsset,
+  IAssetMapById,
+  getAssetsMaps,
+  isDevMode,
+} from '@src/utils';
```

Comment:
on this file, add log:

`console.log(`[STORAGE] Refresh quote ${new Date()}`);`